backstop_data/engine_scripts/playwright/beforeTakingScreenshotHelper.js

```diff
@@ -0,0 +1,58 @@
+module.exports = async (page, { beforeTakingScreenshot }) => {
+  const allowedPageActions = {};
+  const postInteractionWait = 250; // ms
+
+  allowedPageActions.click = async selector => {
+    await page.locator(selector).click();
+    await page.waitForTimeout(postInteractionWait);
```

Comment:
in the end, I don't know if these waitForTimeout are overkill

it's only the account header examples that seem to need a delay - but then they need much more than 250ms but I think 250ms probably isn't costing much for a bit of extra smoothing over of stuff? feels bad but 🤷 

just seems to be something in screenshotting hover/focus that's unreliable

my idea for debugging further would be to make a recording of just the scenario in question using playwright and see how state changes over time while scenario runs

but not sure it's worth it if the 2s delay covers it